cinema_service/settings.py

```diff
@@ -20,12 +20,11 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = (
-    "django-insecure-ru6ndfka@95_(lysua8yhdjq@vpiqgv3yru4r)q3h4_u8x7dfy"
-)
+SECRET_KEY = ("django-insecure-ru6ndfka@95_(l"
+              "ysua8yhdjq@vpiqgv3yru4r)q3h4_u8x7dfy")
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = False
```

Comment:
Ensure that `ALLOWED_HOSTS` is configured with the domain names or IP addresses that your application will serve when `DEBUG` is set to `False`.